dotnet/src/Extensions/PromptTemplates.Handlebars/HandlebarsPromptTemplate.cs

```diff
@@ -48,7 +48,7 @@ public async Task<string> RenderAsync(Kernel kernel, KernelArguments? arguments
         this.RegisterHelpers(handlebarsInstance, kernel, arguments, cancellationToken);
 
         var template = handlebarsInstance.Compile(this._promptModel.Template);
-        return template(arguments).Trim();
+        return template(arguments).Trim().Replace("&quot;", "\"");
```

Comment:
Why is just this one HTML entity being handled?

Should this use `System.Net.WebUtility.HtmlDecode()`

Reply:
It was just the one I noticed the most

Good call, will change to `System.Net.WebUtility.HtmlDecode(template(arguments).Trim());`